physical/postgresql/postgresql.go

```diff
@@ -6,31 +6,76 @@ import (
 	"fmt"
 	"strconv"
 	"strings"
+	"sync"
 	"time"
 
 	"github.com/hashicorp/errwrap"
 	"github.com/hashicorp/vault/physical"
 
 	log "github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/go-uuid"
 
 	metrics "github.com/armon/go-metrics"
 	"github.com/lib/pq"
 )
 
+const (
+
+	// (seconds) The lock TTL matches the default that Consul API uses, 15 seconds.
+	// used in as part of SQL commands to set/extend lock expiryt time relative to database clock
+	PostgreSQLLockTTL = 15
```

Comment:
Since this const is untyped maybe append "Seconds" to the name?

Reply:
yes I agree makes it much more readable.